conversation/frontend/src/features/Message/message-list.tsx

```diff
@@ -90,6 +101,11 @@ export function MessageList() {
     if (event.key === ' ' || event.key === 'Enter') {
       handleMessageClick(message);
     }
+  }
+
+  const handleMoveToTrash = () => {
```

Comment:
```suggestion
  const handleMoveToTrashClick = () => {
```
Pour respecter les conventions de nommage en place